### ISSUE TITLE
[2.3.0] Object labels (M486) cannot be turned off

### ISSUE THREAD

**Issue by kamilkrz** [SoftFever/OrcaSlicer#7775]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.3.0

### Operating System (OS)

Linux

### OS Version

Distributor ID: Linux mint  Release: 21.1

### Additional system information

Exists in: [OrcaSlicer_Linux_V2.3.0-dev.AppImage](https://github.com/SoftFever/OrcaSlicer/releases/download/nightly-builds/OrcaSlicer_Linux_V2.3.0-dev.AppImage)

### Printer

Artillery genius pro

### How to reproduce

1. Be sure to turn off "label objects" and "exclude objects"
2. Slice print
3. Check gcode for M486

### Actual results

![image](https://github.com/user-attachments/assets/7b0b3e0e-3594-4d34-835e-14c265a14e67)

### Expected results

M486 obj_name
M486 S0
M486 S -1
Shoudld be not present

### Project file & Debug log uploads

[benchy.zip](https://github.com/user-attachments/files/18170306/benchy.zip)

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by tlhintoq** [comment]
**### One more problem with this bug...**
These `M486` commands are being adding to the Neptune 3 Max jobs - and that's a Marlin machine not Klipper.
This is resulting in the job rebooting the Neptune 3 Max - It just doesn't know what to do with these commands.
![2025-01-04_12-11-08](https://github.com/user-attachments/assets/6312ab7a-5f41-4672-9cfc-a1648552386b)

As soon as I comment the `M486` block - the job runs fine.

![2025-01-04_12-21-38](https://github.com/user-attachments/assets/38f064e5-fce0-4cfe-b7de-d7fefa938ee7)

**Comment by Rsoft1976** [comment]
yes there is, I confirm, the M486 error constantly pops up, it is not enabled in the settings but it shows
